index.md

```diff
@@ -89,19 +89,56 @@ as to mitigate subjectivity in role assignment.
 
 ### [Infractions](#infractions)
 It is incumbent on members to bring issues to the attention of the community.
-An issue can't be addressed if no one else is aware it exists. Either bring up
-the issue directly with the other member or speak to some of the other members
-that you are close with. If they agree that it's an issue that needs
-addressing, the matter should be brought to the community as a whole for
-discussion.
-
-### [Sanctions](#sanctions)
-* Apologize when appropriate
-* Ameliorate the situation (e.g. repair or replace a damaged game; buy person a
-  lunch to mend things)
-
-Continued violations may result in ejection from the community, by process of
-consensus of active and long-standing members.
+An issue can't be addressed if no one else is aware it exists. Ideally, members
+should bring up the issue directly with the other member, solving it privately.
+The next recourse is to initiate a formal conflict resolution proceeding.
+Before doing so, it is highly recommended that a member speak with one or two
+other members about the matter.  If they agree that it's an issue that needs
+addressing, the matter should be brought to the attention of the
+[Adepts](#contributing-members) and will move through the conflict resolution
+procedure described below.  Note that it is not necessary to relay the
+particulars of a conflict to the Adepts in order to begin conflict resolution.
+
+### Conflict Resolution
+
+#### [Mediation](#mediation)
+Conflict resolution will initially begin with a mediation phase.  A mediator
+will be selected from the [Adept](#contributing-members) group either by mutual
+agreement between the two conflicting parties or via selection of a
+disinterested party by the Adepts.
+
+The mediator will be responsible for facilitating a discussion between the two
+parties, attempting to ascertain the facts behind the conflict, and for
+proposing a solution amicable to both groups.  Mediation is expected to remain a
+private matter between the two parties and the mediator.  If the two parties are
+still unable to reach an accord, the process moves to arbitration.
```

Comment:
```suggestion
The mediator facilitates a discussion between the two
parties, attempting to gather facts behind the conflict, and for
proposing a good-natured solution to both groups.  Mediation is expected to remain a
private matter between the two parties and the mediator.  If the two parties are
still unable to reach a solution, the process moves to arbitration.
```

Reply:
```
- solution amicable
+ good-natured solution
```
I think this changes the meaning. Alternate versions that don't would be along the lines of `a solution agreeable to both groups` or `a solution both members can agree to`.

Reply:
That's reasonable. Of you're two suggestions I like `a solution agreeable to both groups`.

Reply:
Thinking more, I believe "amicable" would be fine.

```
proposing an amicable solution.`

"Amicable" is already referring to relations between people, so I believe "to both groups" would be redundant.

Reply:
`an amicable solution` sounds good to me.